### PR TITLE
Added the telegram notifier

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added the telegram notifier

### CODE REVIEW

File: notifiers/telegramNotifier_test.go [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+const testTelegramToken = "test-token"
+const testTelegramChatID = "test-chat-id"
+
+func createHttpTestServerThatRespondsOKForTelegram(
+	t *testing.T,
+	expectedMessage string,
+	expectedTitle string,
+	numCalls *uint32,
+) *httptest.Server {
+	response := &pushoverResponse{
```
Comment (andreibancioiu) [inline notifiers/telegramNotifier_test.go:27]: This response object is specific to pushover. Maybe use one for TG?

**Review by miiu96 — DISMISSED** [review]

File: notifiers/telegramNotifier.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+	return nil
+}
+
+// IsInterfaceNil returns true if there is no value under the interface
+func (notifier *telegramNotifier) IsInterfaceNil() bool {
+	return notifier == nil
+}
+
+// https://api.telegram.org/bot7187383001:AAFKmIYFFVF3OLi7ibEE_V2KV0C1k2KGOng/sendMessage?chat_id=589289766&parse_mode=html&text=<b>aaa</b>
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply notifiers/telegramNotifier.go:82]: indeed an example, removed

File: notifiers/telegramNotifier_test.go (hunk 1 above)
Comment (iulianpascalau) [reply notifiers/telegramNotifier_test.go:27]: copy-paste error. Removed

File: notifiers/telegramNotifier.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+	return &telegramNotifier{
+		httpClientWrapper: httpSDK.NewHttpClientWrapper(nil, url),
+		token:             token,
+		chatID:            chatID,
+	}
+}
+
+// OutputMessages will push the provided messages as error
+func (notifier *telegramNotifier) OutputMessages(messages ...core.OutputMessage) {
+	log.Debug("notifier.OutputMessages sending telegram notification", "num messages", len(messages))
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply notifiers/telegramNotifier.go:29]: refactored all log messages

**Review by andreibancioiu — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

**Pull request merged**